### PR TITLE
Add B2B endpoints

### PR DESCRIPTION
**Pull request by logan-stytch**
Allll the codegen for the new B2B endpoints!

For testing, I ran the following actions locally:
```
organizations.create
organizations.get
organizations.update
organizations.delete
organizations.search
organizations.search_members (both with and without a supplied query)

magic_links.authenticate
magic_links.email.login_or_signup
magic_links.email.invite

sessions.get

```
For `sessions.authenticate` and `sessions.revoke`, I verified the protobuf signature matched, but didn't test with a real sessions since I don't have a project fully set up.

### CODE REVIEW

**Review by vincent-stytch — DISMISSED** [review]
we might be in the process of changing these

File: codegen/specs/stytch_b2b/docs/magic_links/authenticate.response.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+- `organization_id`: Globally unique UUID that identifies a specific Organization in the Stytch API. The organization_id is critical to perform operations on an Organization so be sure to preserve this value.
+
+- `member`: The Member object.
+
+- `session_token`: A secret token for a given Stytch Session. Read more about session_token in our Session management guide.
+
+- `session_jwt`: The JSON Web Token (JWT) for a given Stytch Session. Read more about session_token in our Session Management guide.
+
+- `session`: The B2BStytchSession object.
```
Comment (jhaven-stytch) [inline codegen/specs/stytch_b2b/docs/magic_links/authenticate.response.md:17]: I'm not sure if this is a difference between hitting the api directly vs web-backend (which the SDKs hit), but in the SDK response, this is called `member_session`

File: codegen/specs/stytch_b2b/docs/organizations/search.request.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Search for Organizations. If you send a request with no body params, no filtering will be applied and the endpoint will return all Organizations.
+
+All fuzzy search filters require a minimum of at least 3 characters
+
+Parameters:
+
+- `cursor`: The cursor field allows you to paginate through your results. Each result array is limited to 1000 results, if your query returns more than 1000 results, you will need to paginate the responses using the cursor. If you receive a response that includes a non-null next_cursor in the results_metadata object, you should repeat the call, being sure to include all of the original fields, but pass in the next_cursor in the cursor field. Continue to make calls until the next_cursor in the response is null.
+
+- `limit`: The number of Organizations to return per page, the default is 100. A maximum of 1000 Organizations can be returned by a single request. If the total size of your result is greater than one page size, you must paginate the response. See the cursor field below.
```
Comment (jhaven-stytch) [inline codegen/specs/stytch_b2b/docs/organizations/search.request.md:9]: `See the cursor field below` should say `above`

File: codegen/specs/stytch_b2b/docs/organizations/search_members.request.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Search for members within an Organization. It requires an organization_id. Submitting an empty query returns all Members of an Organization.
+
+Parameters:
+
+- `organization_ids`: List of globally unique UUIDs that identify specific Organizations. The organization_id is critical to perform operations on an Organization, so be sure to preserve this value.
+
+- `cursor`: The cursor field allows you to paginate through your results. Each result array is limited to 1000 results, if your query returns more than 1000 results, you will need to paginate the responses using the cursor. If you receive a response that includes a non-null next_cursor in the results_metadata object, you should repeat the call, being sure to include all of the original fields, but pass in the next_cursor in the cursor field. Continue to make calls until the next_cursor in the response is null.
+
+- `limit`: The number of Organizations to return per page, the default is 100. A maximum of 1000 Members can be returned by a single request. If the total size of your result is greater than one page size, you must paginate the response. See the cursor field below.
```
Comment (jhaven-stytch) [inline codegen/specs/stytch_b2b/docs/organizations/search_members.request.md:9]: same nit re: below/above

File: codegen/specs/stytch_b2b/magic_links.yml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        arg_type: Optional[Dict[str, Any]] = None
+    response_type:
+      member_id: str
+      method_id: str
+      reset_sessions: bool
+      organization_id: str
+      member: Member
+      session_token: str
+      session_jwt: str
+      session: Optional[B2BStytchSession]
```
Comment (jhaven-stytch) [inline codegen/specs/stytch_b2b/magic_links.yml:26]: same question re: `session` vs `member_session`

File: stytch/b2b/models/magic_links.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    """  # noqa
+
+    member_id: str
+    method_id: str
+    reset_sessions: bool
+    organization_id: str
+    member: Member
+    session_token: str
+    session_jwt: str
+    session: Optional[B2BStytchSession]
```
Comment (jhaven-stytch) [inline stytch/b2b/models/magic_links.py:36]: `session` vs `member_session`

File: stytch/b2b/core/models.py [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    email_allowed_domains: List[str]
+    email_jit_provisioning: str
+    email_invites: str
+
+
+class SSORegistration(pydantic.BaseModel):
+    connection_id: str
+    identity_provider: str
+    # TODO: Should this be external_id? (missing l)
+    externa_id: str
```
Comment (jhaven-stytch) [inline stytch/b2b/core/models.py:29]: Pointing out the TODO, was this updated?

**Review by jhaven-stytch — APPROVED** [review]
I think this looks good! Just a few nits in the documentation, and a lingering question/uncertainty about `session` vs `member_session` naming

File: codegen/specs/stytch_b2b/docs/magic_links/authenticate.response.md (hunk 1 above)
Comment (logan-stytch) [reply codegen/specs/stytch_b2b/docs/magic_links/authenticate.response.md:17]: Good catch, I updated the API definitions, but not the docs.

Comment (logan-stytch) [reply codegen/specs/stytch_b2b/docs/magic_links/authenticate.response.md:17]: Actually I see this one was completely missed now :O

**Pull request merged**